js/src/util.js

```diff
@@ -189,6 +189,10 @@ const Util = (() => {
       return false
     },
 
+    noop() {
+      return null
```

Comment:
`noop` should return an empty function

Reply:
`noop` should be an empty function, but ESLint does not allow empty functions

Reply:
Disable that rule just for this line ;)

Reply:
Done